esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -306,8 +316,8 @@ define([
                 arrayUtil.forEach(this.desprayGrid.store.data, function (item, idx) {
                     var request = domForm.toObject(context.id + "DesprayForm");
                     request.destPath = context.desprayTargetPath.getDropZoneFolder();
-                    if (!context.endsWith(request.destPath, "/")) {
-                        request.destPath += "/";
+                    if (!context.endsWith(request.destPath, request.destPath)) {
+                        request.destPath += request.destPath;
```

Comment:
So previously, we were hard coding "/" after issuing different breakpoints throughout the code I realized that request.destPath always returned "/" or "\" appropriately for the given path for the environment that was being used for that machine.